contracts/lootbox-v1/test/sample-test.js

```diff
@@ -0,0 +1,19 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+describe("Greeter", function () {
+  it("Should return the new greeting once it's changed", async function () {
+    const Greeter = await ethers.getContractFactory("Greeter");
+    const greeter = await Greeter.deploy("Hello, world!");
+    await greeter.deployed();
+
+    expect(await greeter.greet()).to.equal("Hello, world!");
+
+    const setGreetingTx = await greeter.setGreeting("Hola, mundo!");
+
+    // wait until the transaction is mined
+    await setGreetingTx.wait();
+
+    expect(await greeter.greet()).to.equal("Hola, mundo!");
+  });
+});
```

Comment:
we don't need this file

Reply:
Thanks for your reviewing and it was great help for me.
Regarding about compile issue I mentioned above.
And other things were on prgoress of implementing the WyvernExchange.
I need to convert it to Genie now and from now I am sure I will send PR when it complies 100%